t/ui/15-admin-workers.t

```diff
@@ -32,6 +32,8 @@ my $broken_worker_id  = 5;
 my $online_worker_id  = 6;
 my $offline_worker_id = 8;
 
+my $undef_var;
+
```

Comment:
This variable doesn't seem to be used after all.

Reply:
Interesting. Who put that in :D

Reply:
fixed